lib/developer_portal/lib/cms/toolbar.rb

```diff
@@ -17,7 +17,7 @@ def cms_toolbar
   protected
 
   def handle_cms_token
-    token = params.delete(:cms_token)
+    token = validate_and_extract_cms_token
 
     if cms.valid_token?(token)
```

Comment:
here basically you wouldn't care about the token value, but something like `if cms.account_id == signed_account_id`

Or whatever other convenient CMS identification value is used to generate the signature.

Reply:
Same as here: https://github.com/3scale/porta/pull/3938#discussion_r1832297257